dashboard/src/components/TreeListingPage/TreeTable.tsx

```diff
@@ -71,11 +71,7 @@ const columns: ColumnDef<TreeTableBody>[] = [
               {sanitizeTableValue(row.getValue('tree_name') ?? '', false)}
             </div>
           </TooltipTrigger>
-          <TooltipContent>
-            <a href={row.original.url} target="_blank" rel="noreferrer">
-              {sanitizeTableValue(row.original.url, false)}
-            </a>
```

Comment:
I don't see a reason to remove this link, it seems that it is possible to open it in a new tab even if clicking on the cell will direct you to the treeDetails page